src/main/java/org/sefglobal/scholarx/controller/admin/ProgramController.java

```diff
@@ -55,9 +56,15 @@ public void deleteProgram(@PathVariable long id) throws ResourceNotFoundExceptio
 
     @GetMapping("/{id}/mentors")
     @ResponseStatus(HttpStatus.OK)
-    public List<Mentor> getAllMentorsByProgramId(@PathVariable long id,
-                                                 @RequestParam(required = false) List<EnrolmentState> states)
+    public List<Mentor> getAllMentorsByProgramId(@PathVariable long id, @RequestParam(required = false) List<EnrolmentState> states)
             throws ResourceNotFoundException {
         return programService.getAllMentorsByProgramId(id, states);
     }
+
+    @GetMapping("/{id}/mentees")
+    @ResponseStatus(HttpStatus.OK)
+    public List<Mentee> getAllMenteesByProgramId(@PathVariable long id)
```

Comment:
Can you create the unit test for this ?

Reply:
Let's complete the testing on a later stage @anjula-sack create an issue for this.

Reply:
sure